client/client.go

```diff
@@ -81,8 +82,9 @@ func New(config Config) (*Client, error) {
 }
 
 //Put create value of a key
-func (c *Client) Put(ctx context.Context, kv model.KVDoc) (*model.KVDoc, error) {
-	url := fmt.Sprintf("%s/%s/%s", c.opts.Endpoint, APIPathKV, kv.Key)
+func (c *Client) Put(ctx context.Context, kv model.KVDoc, project string) (*model.KVDoc, error) {
+	finalPath := fmt.Sprintf("%s/%s/%s", version, project, APIPathKV)
+	url := fmt.Sprintf("%s/%s/%s", c.opts.Endpoint, finalPath, kv.Key)
```

Comment:
fmt.Sprintf is graceful, but it has poor performance, never use it in  func that handles a lots of data